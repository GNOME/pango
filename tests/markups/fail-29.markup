<span overline-color="rgb(1,2,3)">test</span>
