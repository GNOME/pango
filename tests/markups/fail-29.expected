ERROR: Value of 'overline_color' attribute on <span> tag on line 1 could not be parsed; should be a color specification, not 'rgb(1,2,3)'